### ISSUE TITLE
Doesn't work w/ npx

### ISSUE THREAD

**Issue by zephraph** [intuit/auto#244]
**Describe the bug**

Running `npx auto shipit` results in an error. 

```
Error: Cannot find module './dist/cjs'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/zephraph/.npm/_npx/292/lib/node_modules/auto/node_modules/parse-commit-message/index.js:4:13)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
```

**To Reproduce**

Run `npx auto shipit`

**Comment by zephraph** [comment]
Okay, so my debugging so far has shown that the package `parse-commit-message` [requires esm](https://github.com/tunnckoCoreLabs/parse-commit-message/blob/93018525c85296b44b41ddb7af1700afd5a54d60/index.js#L4) which seems to fail when used in the context of npx. I'm not certain why that's the case, but it seems to be so. I'll keep digging.

**Comment by hipstersmoothie** [comment]
I’m at the gym. Are we on the beta release of parse commit message? The
creator should have removed it in v4

**Comment by hipstersmoothie** [comment]
(Esm)

**Comment by zephraph** [comment]
No rush, finish your workout! I'll keep digging.

**Comment by aleclarson** [comment]
Should probably add a test for `npx auto`, which would've caught this

**Comment by zephraph** [comment]
Interesting. 

Here's two failing CI builds:
https://circleci.com/gh/artsy/reaction/24087
https://circleci.com/gh/artsy/palette/2479

And it failing on my (windows) machine

![image](https://user-images.githubusercontent.com/3087225/51586188-3ab93280-1eaa-11e9-87c8-29baecfc9b7f.png)

I haven't tried it on mac yet. Running Node 10 on all of these.

**Comment by hipstersmoothie** [comment]
I was using npx without --ignore-existing  so i was getting old code. Now it's broken

**Comment by hipstersmoothie** [comment]
We should be using a version of parse-commit-message that doesn't use esm

**Comment by zephraph** [comment]
Yeah, so I was wrong. . I was just seeing `parse-commit-message` fail on line 4. [This](https://github.com/tunnckoCore/monorepo/blob/a2bf13527da064404aa76b21893180520d01d6d4/packages/parse-commit-message/index.js#L4) makes more sense. It's the same as the error.

**Comment by hipstersmoothie** [comment]
It probably just needs to be `dist/cjs/index` ?

**Comment by zephraph** [comment]
I would think the index would be implicit..

**Comment by hipstersmoothie** [comment]
where does npx actually install things?

**Comment by zephraph** [comment]
Okay, so, I just tested a hunch. If you install it via npm (even locally) not all the files are there.

**Comment by hipstersmoothie** [comment]
I did write tests for it and they passed 😕

**Comment by zephraph** [comment]
Okay, v4.0.0 has all the files v4.0.1 is missing dist.

**Comment by hipstersmoothie** [comment]
ah

**Comment by zephraph** [comment]
This will be temporarily fixed by #245. Hopefully.
